scripts/rollup/index.js

```diff
@@ -2,7 +2,7 @@ import reactRouter from "./react-router.js";
 import reactRouterDOM from "./react-router-dom.js";
 import reactRouterNative from "./react-router-native.js";
 
-export default function (options) {
+export default function rollup(options) {
```

Comment:
New react-app eslint config warns about anonymous default exports. Probably not a big deal for these files but this was simple enough to keep it happy!